data-science-pipelines-operator/samples/custom-configs/db-creds.yaml

```diff
@@ -0,0 +1,10 @@
+kind: Secret
+apiVersion: v1
+metadata:
+  name: testdbsecret
+  labels:
+    app: mariadb-sample
+    component: data-science-pipelines
+stringData:
+  password: testingpassword
```

Comment:
Is this password modified after? If not, it could be a security problem

Reply:
removed the samples directory